src/components/Crypto/styles.tsx

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+
+export const CryptoContainer = styled.View`
+    alignSelf:center;
+    height: 100px;
+    width: 85%;
+    marginBottom: 10px;
+    borderBottomColor: #C6C8D9;
+    borderBottomWidth:1px
+`
+
+export const NameContainer = styled.View`
+    width: 25%;
+    position: relative;
+    left: 22%;
+    top: 20%;
+`
```

Comment:
in this file there are a feew semicolons missing. Also remember to leave one line between components